app/routes/__frontend/resources/developer-resources/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+meta:
+  title: Developer Resources
+  description: A collection of resources for all developers.
```

Comment:
What's the difference between developer tips and developer resources? Where do these live on the site?